src/Command/InceptionResult.php

```diff
@@ -84,7 +86,7 @@ public function getGenerateBaselineFile(): ?string
 		return $this->generateBaselineFile;
 	}
 
-	public function handleReturn(int $exitCode, ?int $peakMemoryUsageBytes): int
+	public function handleReturn(int $exitCode, ?int $peakMemoryUsageBytes, ?float $analysisStartTime): int
```

Comment:
By looking at the diff it seems to me that `null` is never passed here so we can just have `float`, right?

Reply:
Yeah, lets get rid of that `?`